sale_order_pending_move_menu/models/stock_move.py

```diff
@@ -0,0 +1,16 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models, fields, api
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    move_sale_order = fields.Many2one("sale.order", related="sale_line_id.order_id", store=True, string="Order id")
```

Comment:
Better field naming and label:
```suggestion
    move_sale_order_id = fields.Many2one("sale.order", related="sale_line_id.order_id", store=True, string="Sale Order")
```